static_src/components/header.jsx

```diff
@@ -0,0 +1,60 @@
+
+import React from 'react';
+
+import createStyler from '../util/create_styler';
+
+import headerStyle from 'cloudgov-style/css/components/header.css';
+import navStyle from 'cloudgov-style/css/components/nav.css';
+import logoStyle from 'cloudgov-style/css/components/logo.css';
+
+export default class Header extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.styler = createStyler(headerStyle, navStyle, logoStyle);
+  }
+
+  getImagePath(iconName) {
```

Comment:
Possibly. I think some components will include similar, if not the same, CSS files but that shouldn't be a problem with CSS modules.

I tried to get the `createStyler` function to be most of the re-usable parts but we can always refactor again later if see a common pattern emerge